src/components/keymap/header/keymap-header.component.html

```diff
@@ -1,13 +1,17 @@
 <div class="row">
     <h1 class="col-xs-12 pane-title">
         <i class="fa fa-keyboard-o"></i>
-        <input class="keymap__name pane-title__name"
+        <input #name class="keymap__name pane-title__name"
+               type="text"
                value="{{ keymap.name }}"
                (change)="editKeymapName($event.target.value)"
+               (keyup.enter)="name.blur()"
```

Comment:
Shouldn't we use renderer with `invokeElementMethod` here? I don't know. I don't have information about is it accepted like this in the template or not.